README.org

```diff
@@ -613,6 +614,8 @@ controlled by the ~:multi-param-style~ option:
 ;; with :multi-param-style :indexed, a repeating param with array suffix and
 ;; index (Rails-style):
 :a [1 2 3] => "a[0]=1&a[1]=2&a[2]=3"
+;; with :multi-param-style :enumerated, a param with enumerated values
```

Comment:
I would suggest naming this to `:comma-separated` to describe precisely the behaviour.

Reply:
Totally agree.